hw/ip/rstmgr/data/rstmgr.sv.tpl

```diff
@@ -321,4 +321,13 @@ module rstmgr import rstmgr_pkg::*; (
 
   // when upstream resets, downstream must also reset
 
+  // output known asserts
```

Comment:
I think it might be better to have a dedicated commit for the milestone updates.